k2/python/k2/rnnt_loss.py

```diff
@@ -1019,7 +1019,7 @@ def rnnt_loss_smoothed(
     boundary: Optional[Tensor] = None,
     reduction: Optional[str] = "mean",
     return_grad: bool = False,
-) -> Tensor:
+) -> Union[Tuple[Tensor, Tuple[Tensor, Tensor]], Tensor]:
```

Comment:
I think current return type is right, `rnnt_loss_smoothed` may return grads if return_grad equals True.